lexicons/com/atproto/admin/updateCommunicationTemplate.json

```diff
@@ -0,0 +1,49 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.updateCommunicationTemplate",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Administrative action to update an an existing communication template.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["id", "updatedBy"],
```

Comment:
hmm.... I don't expect anyone else to be using these apis but making them option means we should throw an error from the handler if these are missing until auth call fill it in?

Reply:
> I don't expect anyone else to be using these apis

I think we expect others to run their own services that use these same APIs.  We can coordinate releases for ourselves but it's not as easy to do for others, so I think if it's easy to avoid breaking lexicon changes that would be ideal!

> making them option means we should throw an error from the handler if these are missing until auth call fill it in?

Yeah, I think so!  That gives us the same behavior for ourselves but is more flexible at the lexicon level, where we know/expect we'll make use of it.